src/App.css

```diff
@@ -12,8 +11,9 @@
   background: #1d1e22;
 }
 
-.custom-word-break {
-  word-break: break-all;
+.code-mirror-custom {
```

Comment:
Why do you need to use the !important flag here? Can this not be handled in the regular flow?

Reply:
I tried the regular flow at first but the since the original styling is inherited from the parent class "CodeMirror-line", the child inherits the property and completely ignores my custom-word-break class.